scripts/g.extension/g.extension.py

```diff
@@ -675,8 +679,43 @@ def install_extension(source, url, xmlurl):
         if sys.platform == "win32":
             ret += install_extension_win(module)
         else:
+            # set /usr/bin/python to python3 cause of python shebangs
+            ps = gscript.Popen((
+                "update-alternatives", "--install", "/usr/bin/python",
+                "python", "/usr/bin/python3", "1"))
+            # install extension
             ret1, installed_modules, tmp_dir = install_extension_std_platforms(module,
                                                    source=source, url=url)
+
+            # change python to python3 in shebang
+            proc = subprocess.Popen(
+                "which " + module, shell=True,
```

Comment:
`which` cannot be used since it's not multiplatform

Reply:
Also the which is removed